connector/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -17,365 +17,41 @@
 
 package org.apache.spark.sql.avro
 
-import java.util.Locale
+import org.apache.avro.Schema
 
-import scala.collection.mutable
-import scala.jdk.CollectionConverters._
+import org.apache.spark.annotation.Evolving
+import org.apache.spark.sql.avro.{SchemaConverters => CoreSchemaConverters}
 
-import org.apache.avro.{LogicalTypes, Schema, SchemaBuilder}
-import org.apache.avro.LogicalTypes.{Date, Decimal, LocalTimestampMicros, LocalTimestampMillis, TimestampMicros, TimestampMillis}
-import org.apache.avro.Schema.Type._
+@deprecated("Use SchemaConverters in sql/core instead", "4.0.0")
+@Evolving
+object SchemaConverters {
```

Comment:
@gengliangwang I know you wanted to keep a deprecated file with the same name here, but it conflicts with the SchemaConverters file we have moved to sql/core.